### PR TITLE
replace the word `hang` with something more appropriate

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Depending on the usage, had to choose different ways of expressing it.  Didn't change use of `hanging` for formatting as that term is specific.

Fix https://github.com/PowerShell/PowerShell/issues/5147

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -447,7 +447,7 @@ For more information on this, we invite you to read [this blog post explaining P
 
 ### Remoting
 
-- Fixed hang when the SSH client abruptly terminates. (#4123)
+- Fixed not responding when the SSH client abruptly terminates. (#4123)
```
Comment (TravisEz13) [inline CHANGELOG.md]: the grammar doesn't look right here

**Comment by TravisEz13** [comment]
did you look for `hung` as well?

File: CHANGELOG.md (hunk 1 above)
Comment (SteveL-MSFT) [reply CHANGELOG.md]: missing a noun.  adding `PowerShell`

**Comment by SteveL-MSFT** [comment]
@TravisEz13 fixed 3 instances of `hung`

**Review by TravisEz13 — APPROVED** [review]
I only see minor grammar issues.

**Review by iSazonov — APPROVED** [review]

**Pull request merged**